test/test_time.py

```diff
@@ -83,6 +83,37 @@ def test_rollover(self):
         assert et.bell == 3
         assert et.sun == 4
 
+    def test_negative_rollover(self):
+        et = EorzeaTime()
+        et.sun = 1
+        et.bell = 0
+        et.minute = 0
+
+        et.minute -= 1
+
+        assert et.minute == 59
+        assert et.bell == 23
+        assert et.sun == 32
+        assert et.moon == 1
+        assert et.year == 1070
+
+        et.minute -= 119
+
+        assert et.minute == 0
+        assert et.bell == 22
+
+        et.bell -= 48
+
+        assert et.minute == 0
+        assert et.bell == 22
+        assert et.sun == 30
+        assert et.moon == 1
+
+        et.moon -= 11
+
+        assert et.moon == 2
+        assert et.year == 1069
+
```

Comment:
To ensure the result can be reproduced in future. Please specify timestamp to create EorzeaTime or assign a value to `et.moon` and `et.year`

For example,
This test will be failed when `EorzeaTime(timestamp=datetime(2030, 1, 23).timestamp())`